README.md

```diff
@@ -61,52 +63,55 @@ Review the parameters in [cloudformation.yml](apps/main-cf.yml) for deploy time
 
 ### Deploy with CloudFormation
 
```

Comment:
It might be worth a note/admonishion here to say that manual deployments should only be done to temporary deploymetns, otherwise, they should be hooked into our CI/CD infrastructure... 

I'll think on that a little.

Reply:
more thoughts around this in this comment:
https://github.com/ASFHyP3/hyp3/pull/2566#issuecomment-2613451411